WBooks/Library/BookCell.swift

```diff
@@ -28,6 +28,14 @@ class BookCell: UITableViewCell, NibLoadable {
     func setBook(bookViewModel: BookViewModel) {
         title.text = bookViewModel.title
         author.text = bookViewModel.author
-        cover.image = bookViewModel.cover
+        let url = URL(string: bookViewModel.image)
+        if let url = url {
```

Comment:
You could add this logic to bookViewModel image, so the view just use that value.